src/frontend/src/pages/PIAForm/index.tsx

```diff
@@ -124,7 +124,9 @@ const PIAFormPage = () => {
         },
       );
 
-    if (onIntakePage) {
+    const onNewPiaPage = pathname === buildDynamicPath(routes.PIA_NEW, {});
+
+    if (onIntakePage || onNewPiaPage) {
```

Comment:
whenever there is an if logic that is ambiguous let us add comments to it. This might allow other developers know why this is inserted and what it fixes/or breaks if this condition is removed.

Reply:
sure